client/my-sites/checkout/composite-checkout/composite-checkout.js

```diff
@@ -389,7 +389,7 @@ export default function CompositeCheckout( {
 			! hasDomainRegistration( responseCart ) &&
 			! hasTransferProduct( responseCart );
 		const getIsFieldDisabled = () => isDisabled;
-		const tlds = getAllTlds( domainNames );
+		const tlds = getAllTopLevelTlds( domainNames );
 
 		return (
 			<React.Fragment>
```

Comment:
I cannot comment on the lines below, but it seems like the logic for creating the `RegistrantExtraInfoForm` instances seems quite duplicated. Plus it's hard to see if there are any differences between the TLDs or not.

It should be possible to rewrite it to just loop over the tlds that require extra info and return the extra forms for them. Might be for a different PR, but I think given our plans to keep onboarding ccTLDs we need to make it more scalable in the future. It should also rely on the `getApplicableTldsWithAdditionalDetailsForms` method, instead of hardcoding the specific TLDs.